### PR TITLE
feat: sidebar persona connect status

### PR DESCRIPTION
**Pull request by yanzhihong23**
closes #

### CODE REVIEW

**Comment by yanzhihong23** [comment]
@DimensionDev/qa

**Comment by wenluomask** [comment]
https://mask.atlassian.net/browse/MASK-293

File: packages/mask/src/components/InjectedComponents/ToolboxUnstyled.tsx [hunk 1]
```diff
@@ -94,11 +101,43 @@ export function ToolboxHintUnstyled(props: ToolboxHintProps) {
...
+            if (p.linkedProfiles.get(id)) {
+                connected = true
+            }
+        })
+        return connected
+    }, [personas, lastRecognized, activatedSocialNetworkUI])
+
+    const title = useMemo(() => {
+        return !personas.length ? t('create_persona') : !personaConnected ? t('connect_persona') : walletTitle
+    }, [personas, personaConnected])
```
Comment (guanbinrui) [inline packages/mask/src/components/InjectedComponents/ToolboxUnstyled.tsx:120]: Does it react to language switching even if w/o `t` in the deps list?

**Review by guanbinrui — APPROVED** [review]

**Review by Jack-Works — DISMISSED** [review]

**Review by Jack-Works — APPROVED** [review]

**Review by guanbinrui — APPROVED** [review]

**Comment by wenluomask** [comment]
Checked.

**Pull request merged**